public/emojis.js

```diff
@@ -434,43 +533,56 @@ const emojis = {
   TRAM: "🚡",
   TRAP: "🪤",
   TRASH: "🚮",
+  TREAT: "🍦",
   TREE: "🌳",
-  TREX: "🦖",
+  TRICK: "🎃",
+  TRIM: "💇",
   TROPHY: "🏆",
   TRUCK: "🚙",
-  TRUMPET: "🎺",
   TUBE: "🧪",
   TULIP: "🌷",
+  TURBAN: "👳",
   TURTLE: "🐢",
   TYPE: "⌨️",
   UFOS: "🛸",
-  UNICORN: "🦄",
+  UNICORN: "🦄", // but how?
```

Comment:
why include a seven?

Reply:
It's very elusive.

If you can find a 4-6 letter alternative, I'm all for it.

Reply:
https://wordassociations.net/en/words-associated-with/Unicorn
`HORN`
`TUSK`
`RARE`
`MYTH`

or maybe something harder to unlock:
`RARITY`
`HORNED`